examples/video/osxVideoRecorderExample/src/ofApp.cpp

```diff
@@ -1,13 +1,8 @@
 #include "ofApp.h"
```

Comment:
Can you just delete this full example? it doesn't really work in any platform anymore so it's just confusing to have it at all

Reply:
Sure, it is much easier.